webmin/postinstall.pl

```diff
@@ -46,8 +46,8 @@ sub module_install
 	}
 
 # Disable trusting SSL certs unless already enabled
-if (!$miniserv{'trust_real_ip'} && !defined($miniserv{'no_trust_ssl'})) {
-	$miniserv{'no_trust_ssl'} = 1;
+if ($miniserv{'trust_real_ip'} == 1 && &get_webmin_version() <= 2.300) {
```

Comment:
There's no need for a version check because we know what version this is, right??

Reply:
If a user installs version 2.300 and then changes this option back to `1`, we don't want it to be reverted upon the release of Webmin 2.301 and etc...

Reply:
Yes but what if the user skips version 2.300 and installs 2.301 ? We cannot rely on checking the version in the post-install script.